api/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const app = express()
+const port = 3000
+
+app.get('/', (req, res) => {
+    res.send('Banking Platform version 1.0!')
```

Comment:
We should change the version.
```suggestion
    res.send('Banking Platform version 0.0.1!')
```